spec/interactors/api/filter_events_spec.rb

```diff
@@ -3,9 +3,11 @@
 
 RSpec.shared_examples 'filtered by params' do
   subject(:sql) { events.sql }
+
   describe 'deleted record' do
-    it 'is filtered out by default' do
+    it 'filters cancelled and deleted events by default' do
```

Comment:
I’ve tried to improve it, please look at 3cf94f041766ff543e02ebd9d302caad6b26e74a.